src/main/io/dashboard.c

```diff
@@ -580,6 +583,33 @@ static void showDebugPage(void)
 }
 #endif
 
+static void showBBPage(void)
+{
+    uint8_t rowIndex = PAGE_TITLE_LINE_COUNT;
+    int8_t fileNo;
+    if (isBlackboxDeviceWorking()){
+        switch (blackboxConfig()->device) {
+            case BLACKBOX_DEVICE_SDCARD:
+                fileNo = blackboxGetLogFileNo();
+                if( fileNo > 0){
+                    tfp_sprintf(lineBuffer, "File no: %d", fileNo);
+                }else{
+                    tfp_sprintf(lineBuffer, "Log not started!");
```

Comment:
Would 'not ready yet' be more concise here?

Reply:
done